kitsune/community/utils.py

```diff
@@ -59,10 +68,22 @@ def top_contributors_questions(start=None, end=None, locale=None, product=None,
 
     contribution_buckets = search.execute().aggregations.contributions.buckets
 
-    if not contribution_buckets:
+    deletion_metrics_by_contributor = deleted_contribution_metrics_by_contributor(
+        Answer,
+        start=start,
+        end=end,
+        locale=locale,
+        products=[product] if product else None,
+        limit=limit,
+    )
+
+    if not (contribution_buckets or deletion_metrics_by_contributor):
         return [], 0
 
-    user_ids = [bucket.key for bucket in contribution_buckets]
+    user_ids = list(
+        {bucket.key for bucket in contribution_buckets}
+        | {str(contributor_id) for contributor_id in deletion_metrics_by_contributor.keys()}
+    )
     contributor_group_ids = list(
```

Comment:
This is used later in the `user_hits` query. I wonder if we need to do something like min(len(user_ids), 100) or any number for that matter to make sure that we are not fetching far too many profiles

Reply:
Great idea.